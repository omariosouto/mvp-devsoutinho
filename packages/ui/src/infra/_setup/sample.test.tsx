// TODO: Delete when create another test
import React from 'react';
import styled from 'styled-components';
import { render } from '../test/testUtils';

const Sample = styled.div`
  background: ${({ theme }) => theme.colors.primary.main.color};
`;

function Component() {
  return <Sample>Hi</Sample>;
}

// eslint-disable-next-line no-console
console.log(Component);

describe('sample', () => {
  it('snapshot', () => {
    const { container } = render(<Component />);

    expect(container).toMatchSnapshot();
  });

  it('test', () => {
    expect(true).toBe(true);
  });
});
