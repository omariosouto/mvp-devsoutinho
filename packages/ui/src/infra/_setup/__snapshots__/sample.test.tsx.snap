// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample snapshot 1`] = `
.c0 {
  background: #ff6a00;
}

<div>
  <div
    class="c0"
  >
    Hi
  </div>
</div>
`;
